tests/LinterConfigTest.hack

```diff
@@ -87,6 +93,29 @@ final class LinterConfigTest extends HackTest {
     )->toThrow(\Exception::class, 'is not of the correct type');
   }
 
+  public async function testInvalidConfigForCLI(): Awaitable<void> {
+    list($cli, $_, $stdout, $stderr) =
+      $this->getCLIInDirectory(vec[], self::TEST_DIRECTORY);
+    $exit_code = await $cli->mainAsync();
+    expect($stderr->getBuffer())->toMatchRegExp(
+      re'/A linter threw an exception\\:/',
+    );
```

Comment:
I see you use toMatchRegExp everywhere. If you use toContainSubString you can save yourself (and future readers) from having the reason about escaped regular expression characters.